# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoader.cpp
# Commit: c11cffaded9c
# Full Hash: c11cffaded9cfed53b04e5c7875278e9a9e55396
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r 0527d227c6b6 -r c11cffaded9c dom/script/ModuleLoader.cpp
--- a/dom/script/ModuleLoader.cpp	Fri Feb 18 12:45:52 2022 +0000
+++ b/dom/script/ModuleLoader.cpp	Fri Feb 18 12:45:53 2022 +0000
@@ -5,12 +5,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "ScriptLoader.h"
-#include "ScriptLoadRequest.h"
 #include "ScriptTrace.h"
-#include "LoadedScript.h"
 #include "ModuleLoader.h"
-#include "ModuleLoaderBase.h"
-#include "ModuleLoadRequest.h"
 
 #include "jsapi.h"
 #include "js/ContextOptions.h"  // JS::ContextOptionsRef
@@ -20,6 +16,10 @@
 #include "js/PropertyAndElement.h"  // JS_DefineProperty
 #include "js/Realm.h"
 #include "js/SourceText.h"
+#include "js/loader/LoadedScript.h"
+#include "js/loader/ScriptLoadRequest.h"
+#include "js/loader/ModuleLoaderBase.h"
+#include "js/loader/ModuleLoadRequest.h"
 #include "xpcpublic.h"
 #include "GeckoProfiler.h"
 #include "nsIContent.h"
@@ -32,6 +32,7 @@
 #include "mozilla/LoadInfo.h"
 
 using JS::SourceText;
+using namespace JS::loader;
 
 namespace mozilla::dom {
 